components/button/metadata/tool.yml

```diff
@@ -1,7 +1,8 @@
 name: Tool
-status: Verified
+dnaStatus: Deprecated
```

Comment:
status is now "dnaStatus"?

Reply:
No, they are two separate things, which work together in a very confusing fashion to display whether a component is deprecated or whatnot.

Reply:
thanks for the clarification, all good!